### ISSUE TITLE
Distributions and platforms supported by CVMFS in docs (Trac #1755)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1747]
http://software.icecube.wisc.edu/offline_trunk/metaproject/cvmfs.html#distributions-and-platforms

1. Pretty sure the distributions and platforms info is out of date.

2. "Nearly PORT-less" should be replaced by "I3_PORTS" as above, shouldn't it? Don't you need I3_PORTS (aka ports aka DarwinPorts aka i3ports) for py2-v2? I couldn't actually build IceCube metaprojects "icerec" or 
"simulation" without ports using py2-v2, so that seems a bit misleading to me. I tried doing it without ports, since ports is its own ball of wax, but that just led to more wasted time. Maybe I was missing something obvious, but builds don't work without I3_PORTS, so what's the point in adding the wistful comment here?

Migrated from https://code.icecube.wisc.edu/ticket/1755
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:10", 
    "description": "http://software.icecube.wisc.edu/offline_trunk/metaproject/cvmfs.html#distributions-and-platforms\n\n1. Pretty sure the distributions and platforms info is out of date.\n\n2. \"Nearly PORT-less\" should be replaced by \"I3_PORTS\" as above, shouldn't it? Don't you need I3_PORTS (aka ports aka DarwinPorts aka i3ports) for py2-v2? I couldn't actually build IceCube metaprojects \"icerec\" or \n\"simulation\" without ports using py2-v2, so that seems a bit misleading to me. I tried doing it without ports, since ports is its own ball of wax, but that just led to more wasted time. Maybe I was missing something obvious, but builds don't work without I3_PORTS, so what's the point in adding the wistful comment here?", 
    "reporter": "jlanfranchi", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067190995086", 
    "component": "other", 
    "summary": "Distributions and platforms supported by CVMFS in docs", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-06-22T22:20:25", 
    "milestone": "", 
    "owner": "nega", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160622T22:28:09: david.schultz commented:

> "Nearly PORT-less" is meant to express that besides root/geant4/genie, all the software is built without I3_PORTS.  That's a vast improvement from 90% of software coming from I3_PORTS for py2-v1.  Maybe it doesn't help beginners, so could use some modification.
> 
> Also yes, the metaprojects should build fine without them.  You just don't get a few projects that require those programs.  Low energy is the main user of geant4/genie programs, so you're probably a little biased towards finding them "essential".

**Comment by icecube-trac** [comment]
Trac update at 20160623T14:02:13: 
* olivas changed status from "new" to "accepted"
* olivas changed milestone from "" to "July 2016"
* olivas changed owner from "" to "olivas"
* olivas changed type from "defect" to "cleanup"

**Comment by icecube-trac** [comment]
Trac update at 20160623T14:27:43: 
* nega changed status from "accepted" to "assigned"
* nega changed owner from "olivas" to "nega"

**Comment by icecube-trac** [comment]
Trac update at 20160728T16:22:40: 
* david.schultz commented:

> In changeset 2630:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2630"
> update distributions and platforms supported by cvmfs. closes #1747
> ```


* david.schultz changed status from "assigned" to "closed"
* david.schultz changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190112T00:22:37: nwhitehorn commented:

> In changeset 170298:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170298"
> update distributions and platforms supported by cvmfs. closes #1747
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:10: 
* olivas commented:

> Milestone July 2016 deleted


* olivas changed milestone from "July 2016" to ""
